packages/fast-components-react-msft/src/heading/heading.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import { Foundation, HandledProps, TypeLevel, TypographyTag } from "@microsoft/fast-components-react-base";
+import {
+    AlignHeadingBaseline,
+    HeadingLevel,
+    IHeadingHandledProps,
+    IHeadingManagedClasses,
+    IHeadingUnhandledProps
+} from "./heading.props";
+import Typography from "../typography";
+import { IHeadingClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/* tslint:disable-next-line */
+class Heading extends Foundation<IHeadingHandledProps & IManagedClasses<IHeadingClassNameContract>, IHeadingUnhandledProps, {}> {
+    protected handledProps: HandledProps<IHeadingHandledProps & IManagedClasses<IHeadingClassNameContract>> = {
+        alignBaseline: void 0,
+        level: void 0,
+        managedClasses: void 0,
+        tag: void 0,
+        verticalSpace: void 0
+    };
+
+    /**
+     * Stores HTML tag for use in render
+     */
+    private get tag(): TypographyTag {
+        return TypographyTag[this.props.tag];
+    }
+
+    /**
+     * Stores level for use in render
+     */
+    private get level(): TypeLevel {
+        return TypeLevel[`_${this.props.level}`];
```

Comment:
same comment here - should default to a valid value

Reply:
Yep - this was just a miss